### DISCUSSION TITLE
Thumbnail storage

### DISCUSSION THREAD

**Discussion by darrepac** [photoview/photoview#315] [Q&A and Troubleshooting] [answered]
Hello

Where are the thumbnails stored? I do not see any volume dedicated for that in the docker-compose file...  (neither for the database)

thanks

**Comment by viktorstrate** [comment, marked as answer]
Thumbnails are stored in the media cache located at `PHOTOVIEW_MEDIA_CACHE=/app/cache` and persisted in the `api_cache` docker volume.

https://github.com/photoview/photoview/blob/8cf4e8d779a9377c5d8b9cd125c09dafdbb1607f/docker-compose.example.yml#L28-L36

**Comment by darrepac** [comment]
ok so we cannot define where we want them to be stored? or I am missing something?

**Reply by viktorstrate** [reply]
Yes you can, just change the `api_cache` volume to a bind mount. See https://docs.docker.com/storage/bind-mounts/
